### DISCUSSION TITLE
How to change the Wifi channel properly? I seem to do it wrong...

### DISCUSSION THREAD

**Discussion by lasthopegf** [RaspAP/raspap-webgui#1097] [General]
Hi,
recently and with little efforts I got the RaspAP running as desired - kudos for this nice project :)

Today I tried to change the Wifi channel from 1 to 9. So I opened the web interface and stopped the hotspot. I selected the new channel, saved the hotspot settings and restarted it. But somehow the hotspot won't reappear.

Unfortunately I could not get it working again (changing back to 1 did not help, tried reboots and so on).
So I repeated the entire installation and selected channel 9 from the beginning. Now it works flawlessly on channel 9.
But when I try to reconfigure it to a different channel, it will break again.

I hope there is a better way to change the WIFI channel than repeating the installation? Can someone help me?

p.s.: I'm using RaspAP in WiFi Client AP Mode on a Pi 4 with 8GB and current Pi OS lite.

**Comment by billz** [comment]
What version are you using?

**Comment by lasthopegf** [comment]
The one that I reinstalled today shows "RaspAP v2.8.3". The previous one that I messed up first was a few days old - I did not check the exact version of that one...

**Reply by billz** [reply]
See the note about changing hostapd settings in AP-STA
https://docs.raspap.com/ap-sta/#changing-hostapd-settings

**Comment by lasthopegf** [comment]
OK, thanks. This note seems to be new :)
Maybe it would be a good idea to grey out all options that should not be changed once AP-STA mode has been activated...
